runbot/common.py

```diff
@@ -22,6 +22,21 @@
 dest_reg = re.compile(r'^\d{5,}-.+$')
 
 
+def transactioncache(method):
+    def wrapper(self, *args, **kwargs):
```

Comment:
Always good to use [functools.wraps](https://docs.python.org/3/library/functools.html#functools.wraps)
```suggestion
    @functools.wraps(method)
    def wrapper(self, *args, **kwargs):
```